pkg/platform/model/buildplanner/build.go

```diff
@@ -118,6 +119,11 @@ func processBuildPlannerError(bpErr error, fallbackMessage string) error {
 		if ok && code == clientDeprecationErrorKey {
 			return &response.BuildPlannerError{Err: locale.NewExternalError("err_buildplanner_deprecated", "Encountered deprecation error: {{.V0}}", graphqlErr.Message)}
 		}
+		if status, ok := graphqlErr.Extensions[statusExtensionKey].(float64); ok {
```

Comment:
I think the `ok` here is asserting `(float64)`. Couldn't it panic if the `statusExtensionKey` key doesn't exist?

Reply:
Nope, I tried it in the playground. It doesn't panic.